### PR TITLE
Setup prettier

### PR DESCRIPTION
**Pull request by relekang**

### CODE REVIEW

**Comment by relekang** [comment]
Updated 🙂

File: adapter-tests/unit-tests.js [hunk 1]
```diff
@@ -189,24 +201,27 @@ module.exports = function testAdapter(options) {
...
+          value: { views: [{ time: 1490623474639 }] }
         });
         expect(listener2).toHaveBeenCalledWith({
           key: '/a-key',
-          value: { views: [{ time: 1490623474639 }] },
+          value: { views: [{ time: 1490623474639 }] }
         });
         expect(listener2).toHaveBeenCalledWith({
           key: '/b-key',
-          value: { views: [{ time: 1490623474639 }] },
```
Comment (mxstbr) [inline adapter-tests/unit-tests.js]: Also I just noticed, let's add trailing commas too?

Comment (relekang) [inline adapter-tests/unit-tests.js]: 👌

**Comment by relekang** [comment]
Good idea, just added it.

**Pull request merged**